conda/rapidgenomics_cuda11.viz.yml

```diff
@@ -0,0 +1,42 @@
+channels:
+  - rapidsai-nightly
+  - rapidsai
```

Comment:
What's the reason to use both rapidsai and rapidsai-nightly here?

Reply:
The nightly will fall through to the release channel when something doesn't exist. This is a safeguard for things that we deploy to the nightlies but not the release channel. Unfortunately, the "official" cupy conda package hasn't been deployed with the cuda11 toolkit so RAPIDS deployed their own build, which is/was only in the nightlies. It's not a bad idea to have both as a safeguard for things like this.

Reply:
thanks!